tests/test_handlers.py

```diff
@@ -49,7 +50,34 @@ def _zstd_comppressions_handler(
 register_compressor(".zst", _zstd_comppressions_handler)
 
 LOG_DIR: Path = Path("unittest-logs")
-TIMESTAMP_DELTA_MS: int = 64
+
+ASSERT_TIMESTAMP_DELTA_S: float = 0.256
+LOG_DELAY_S: float = 0.064
+TIMEOUT_PADDING_S: float = 0.512
+
+
+def _try_waitpid(target_pid: int) -> int:
+    """
+    Poll for target_pid to finish by repeatedly sleeping and checking waitpid
+    with WNOHANG. If waitpid has not returned the target_pid after roughly 16s,
```

Comment:
"16s" has a high likelihood of going out of sync with the code. How about "after some delay" or something similar?